bin/helpers/functions.sh

```diff
@@ -4,6 +4,6 @@
 function get_linker_ldflags {
     echo "
         -X 'github.com/mysterium/node/server.mysteriumApiUrl=${MYSTERIUM_API_URL}'
-        -X 'github.com/mysterium/node/communication/nats_discovery.natsServerIp=${NATS_SERVER_IP}'
+        -X 'github.com/mysterium/node/communication/natsdiscovery.natsServerIp=${NATS_SERVER_IP}'
```

Comment:
From effective Go (https://golang.org/doc/effective_go.html#names):
> By convention, packages are given lower case, single-word names; there should be no need for underscores or mixedCaps.

Can we name it just `communucation/discovery`? `natsdiscovery` is very unfriendly and is against effective go.